test/integration/integration_test.go

```diff
@@ -160,6 +160,8 @@ func FeatureContext(s *godog.Suite) {
 	// steps to execute `minishift` commands
 	s.Step(`^Minishift (?:has|should have) state "(Does Not Exist|Running|Stopped)"$`,
 		minishift.shouldHaveState)
+	s.Step(`^Profile (.*) (?:has|should have) state "(Does Not Exist|Running|Stopped)"$`,
```

Comment:
There is no need to have Profile with capital letter, it is not a name as Minishift is. All other step definitions starts with minuscule.

Reply:
True